### PR TITLE
Apigatewayservices

### PR DESCRIPTION
**Pull request by GORA-SAG**

### CODE REVIEW

File: apigatewayservices/postmancollections/Readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#### This Postman collection offers you the sample payloads required for REST APIs supported by API Gateway under different sections.
+
+##### This collection covers:
+
+1. APIs - Includes the sample request and response payloads for REST APIs.
+2. Use cases - Includes the postman runner collection that helps you to achieve different use cases.
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/Readme.md:6]: Just hint an example here

File: apigatewayservices/postmancollections/apis/administration-service/Readme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Administration Service
+
+* API Gateway Administration Service provides interface for you to administer various functions of the API Gateway.  
+
+* The user needs to have different functional privileges to administer different functions.
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/apis/administration-service/Readme.md:5]: different - Say special or simply privilage

File: apigatewayservices/postmancollections/apis/alias-management/Readme.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Alias Management
+
+* Alias Management service allows you to manage aliases in the API Gateway. Any user with the **'Manage aliases'** functional privilege can manage the aliases in API Gateway.
+
+* An alias in API Gateway holds environment-specific property values that can be used in policy routing configurations.
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/apis/alias-management/Readme.md:5]: policy routing?? not enough, provide more

File: apigatewayservices/postmancollections/apis/package-and-plan-management/Readme.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Package and Plan Management
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/apis/package-and-plan-management/Readme.md:1]: Modify Readme.md to README.md. Have it consistent everywhere

File: apigatewayservices/postmancollections/apis/Readme.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# APIs
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/apis/Readme.md:1]: Provide a detailed steps with screenshots of how you import it into postman and use it in one of the READMEs or provide a good link where user can refer for using postman to import the files.

File: apigatewayservices/postmancollections/apis/administration-service/Readme.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Administration Service
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/apis/administration-service/Readme.md:1]: Variable like baseURL are imported properly? Explain how these variables are defined in Postman or provide a link

File: apigatewayservices/postmancollections/usecases/Readme.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+API Gateway exposes rest services which is used to do atomic operations in API Gateway such as Create, Retrieve, Update, and Delete. 
+
+Creating an API from API Gateway looks easy with the UI. But the process consist of multiple rest calls that is orchestrated at the backend.
+
+Use cases consist of such type of orchestration which will enable the customer to automate different types of processes.
+
+#### Postman Environment Variable
+
+Retrieve the required variables and set it to postman environment variables for further use under tests tab.
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/usecases/Readme.md:11]: Tests tab

Comment (ClenchPaign) [inline apigatewayservices/postmancollections/usecases/Readme.md:11]: Provide a screenshot of doing this

File: apigatewayservices/postmancollections/usecases/api-application-association/Readme.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+Associate Application with API
+
+##### Step 8:
+
+Activate API
+
+#### Output:
+
+By running this postman collections using postman runner, we will get an API in activated state ready to be invoked with the above configurations.
```
Comment (ClenchPaign) [inline apigatewayservices/postmancollections/usecases/api-application-association/Readme.md:39]: Provide a screenshot of the api invocation in the postman

**Comment by vallabbharath** [comment]
Hi Gokul,

There are conflicts in all swagger files present in webmethods-api-gateway/apigatewayservices/ folder. I think you need to pull this folder from master. Can you please do that?

Regards,
Vallab

**Pull request merged**